3377 isogenity audits

cricketsloan opened the pull request:
This is a total redo of the replication type.  The audits are being removed and a facet for isogenity is added.

lrowe commented:
Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

cricketsloan commented:
I will check it out.


On Nov 18, 2015, at 5:14 PM, Laurence Rowe <notifications@github.com<mailto:notifications@github.com>> wrote:


Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

—
Reply to this email directly or view it on GitHub<https://github.com/ENCODE-DCC/encoded/pull/1024#issuecomment-157915083>.

strattan commented:
Friends don’t let friends say “isogenity” https://github.com/ENCODE-DCC/encoded/pull/1024.

I don’t know if it’s a word or what it might mean.  It sounds vaguely anatomical.

Stick with isogenic, isogenicity  Avoid isogeneic/isogeneity: these have specific meanings in immunology that differ from what we’re trying to say about replicates.  So not audit_experiment_isogeneity

Seth
J. Seth Strattan, PhD --- jseth@stanford.edu<mailto:jseth@stanford.edu> --- www.encodeproject.org<http://www.encodeproject.org> --- 650-725-4735

On Nov 19, 2015, at 8:38 AM, Cricket Alicia Sloan <notifications@github.com<mailto:notifications@github.com>> wrote:


I will check it out.


On Nov 18, 2015, at 5:14 PM, Laurence Rowe <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

—
Reply to this email directly or view it on GitHub<https://github.com/ENCODE-DCC/encoded/pull/1024#issuecomment-157915083>.


—
Reply to this email directly or view it on GitHub<https://github.com/ENCODE-DCC/encoded/pull/1024#issuecomment-158112275>.

cricketsloan commented:
Thanks, Seth.  I have changed my ways and renamed to audit_experiment_isogenic_state.  I will refrain from using isogeneity and any misspellings of that word to avoid the confusion with the immunology definition.

Cricket

On Nov 19, 2015, at 8:43 AM, J. Seth Strattan <notifications@github.com<mailto:notifications@github.com>> wrote:

Friends don’t let friends say “isogenity” https://github.com/ENCODE-DCC/encoded/pull/1024.

I don’t know if it’s a word or what it might mean. It sounds vaguely anatomical.

Stick with isogenic, isogenicity Avoid isogeneic/isogeneity: these have specific meanings in immunology that differ from what we’re trying to say about replicates. So not audit_experiment_isogeneity

Seth
J. Seth Strattan, PhD --- jseth@stanford.edu<mailto:jseth@stanford.edu><mailto:jseth@stanford.edu> --- www.encodeproject.org<http://www.encodeproject.org><http://www.encodeproject.org> --- 650-725-4735

On Nov 19, 2015, at 8:38 AM, Cricket Alicia Sloan <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


I will check it out.


On Nov 18, 2015, at 5:14 PM, Laurence Rowe <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Travis shows this test failing on the branch: test_audit_experiment_model_organism_mismatched_age

—
Reply to this email directly or view it on GitHub<https://github.com/ENCODE-DCC/encoded/pull/1024#issuecomment-157915083>.


—
Reply to this email directly or view it on GitHub<https://github.com/ENCODE-DCC/encoded/pull/1024#issuecomment-158112275>.


—
Reply to this email directly or view it on GitHub<https://github.com/ENCODE-DCC/encoded/pull/1024#issuecomment-158113778>.

lrowe commented on src/encoded/types/experiment.py:
```diff
@@ -273,29 +263,6 @@ def replication_type(self, request, replicates=None):
-            'label': 'Assay',
-        },
-    }
-
```
Looks like you have a merge conflict here. This is part of the matrix branch that got merged recently.

lrowe commented:
Superseded by the 3377-iso-audits-idan branch.
